cloudflare/tests/mf.ts

```diff
@@ -25,6 +23,43 @@ mockAgent
     },
   )
 
+mockAgent
+  .get("http://jsonplaceholder.typicode.com")
+  .intercept({path: "/users/1"})
+  .reply(
+    200,
+    {
+      id: 1,
+    },
+    {
+      headers: {
+        "content-type": "application/json",
+      },
+    },
+  )
+
+mockAgent
+  .get("http://localhost:50051")
+  .intercept({path: "/news.NewsService/GetAllNews", method: "POST"})
+  .reply(
+    200,
+    Buffer.from([
```

Comment:
can we keep it readable string

Reply:
probably possible in some way to define byte string similar to how it's done in `src/grpc/protobuf.rs`. Maybe with `Buffer.from("byte_string", "ascii")`